vision/api/Detect/Program.cs

```diff
@@ -251,6 +256,25 @@ private static object DetectText(Image image)
             return 0;
         }
 
+        private static object DetectTextWithLocation(Image image)
+        {
+            // [START vision_set_endpoint]
+

```

Comment:
Extraneous newline. I looked at the other samples and this isn't present in the other samples.

Reply:
Done.